package.json

```diff
@@ -25,6 +25,7 @@
     "test/esinstall/*"
   ],
   "devDependencies": {
+    "@rollup/plugin-alias": "^3.0.1",
```

Comment:
Is this needed?

Reply:
yea, needed by the Snowpack bundling script